man/man5/zfs-events.5

```diff
@@ -768,7 +773,9 @@ Checksum value is byte swapped.
 \fBbad_ranges\fR
 .ad
 .RS 12n
-Checksum bad offset ranges.
+[start, end) pairs of corruption offsets. Offsets are always aligned on a
```

Comment:
mismatched brackets

Reply:
this is mathematics set notation, not a programming language notation